rules/windows/lateral_movement_incoming_wmi.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies processes executed via Windows Management Instrumentation (WMI) on a remote host. This could be indicative of
+adversary lateral movement but may be noisy if administrators use WMI to remotely manage hosts.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "WMI Incoming Lateral Movement"
+risk_score = 47
+rule_id = "f3475224-b179-4f78-8877-c2bd64c26b88"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan = 5s
+
+ /* Accepted Incoming RPC connection by Winmgmt service */
+
+  [network where event.action == "connection_accepted" and network.direction == "incoming" and
+   not source.address in ("127.0.0.1", "::1") and source.port > 49151 and
```

Comment:
I'm pretty sure that ES doesn't recognize IP fields for `in` right now. I can file a bug for that
```suggestion
   not cidrmatch(source.address, "127.0.0.1", "::1") and source.port > 49151 and
```

Reply:
cidrmatch returned an error, since those are just 2 addresses replaced with 2 "!=" (works on 7.10) 

![image](https://user-images.githubusercontent.com/64742097/99320500-0e835880-286c-11eb-9baa-ca18420135d2.png)